source/features/more-dropdown.tsx

```diff
@@ -43,7 +43,7 @@ function onlyShowInDropdown(id: string): void {
 		return;
 	}
 
-	tabItem!.parentElement!.remove();
+	tabItem!.remove();
```

Comment:
Maybe something like
```suggestion
	tabItem!.closest('[role="tab"]')!.remove();
```
could avoid this next time GH changes its DOM layout.
@fregante thoughts?

Reply:
I had no issues with what you suggested originally, but since you edited your comment, your suggestion is now wrong.
Selecting the closest `.UnderlineNav-item` will cause the issue we have right now.

Reply:
It does work, I wouldn't post something without having tested it first :)

Both my suggestions are equivalent since the targeted element is the same — and is actually `tabItem` itself, so in this case it's a no-op. It's just in case GH decides to wrap the element bearing `[data-tab-item]` again.

Reply:
My bad, sorry, got it confused with `UnderlineNav-body` class.